File: NHSOnline.Backend.Worker/Bridges/Emis/Models/Session.cs
using System.Collections.Generic;

namespace NHSOnline.Backend.Worker.Bridges.Emis.Models
{
    public class Session
    {
        public string SessionType { get; set; }
        public string SessionName { get; set; }
        public int SessionId { get; set; }
        public int LocationId { get; set; }
        public IEnumerable<int> ClinicianIds { get; set; }
    }
}

File: src/main/kotlin/mocking/emis/models/Session.kt
package mocking.emis.models

data class Session(
        var sessionName: String? = null,
        var sessionId: Int? = null,
        var locationId: Int? = null,
        var defaultDuration: Int? = null,
        var sessionType: SessionType? = null,
        var numberOfSlots: Int? = null,
        var clinicianIds: List<Int> = emptyList(),
        var startDate: String? = null,
        var endDate: String? = null
)
